float64[12] leg_lengths
float64 wrist_rot
float64 wrist_flex
